standalone-metastore/pom.xml

```diff
@@ -123,6 +123,13 @@
         <artifactId>orc-core</artifactId>
         <version>${orc.version}</version>
       </dependency>
+      <dependency>
+        <groupId>com.fasterxml.jackson</groupId>
+        <artifactId>jackson-bom</artifactId>
+        <version>${jackson.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
```

Comment:
This needs to be moved to a new `dependencyManagement` section above `dependencies`.

Reply:
Whoops this is in `dependencyManagement`. This means hte jackson-databind dependency below isn't needed.